### ISSUE TITLE
Optional Level Download

### ISSUE THREAD

**Issue by zohnannor** [absolllute/Mega-Hack-Pro-Future#763]
Currently in the game, when you click the "Get it" button, it automatically starts downloading it. Prevent this behavior and render the Download button in the place of a Play button on the level screen.

**Comment by absoIute** [comment]
Why would you want to view the level info page of a level without wanted to play it?

**Comment by zohnannor** [comment]
I agree, use cases for this are pretty niche. For instance when you don't want to play a level but only open its comments or view stats or go to the creator's page. Level loading cannot be aborted and that can be pretty annoying when the level is quite big (or your internet being slow) and you just wanted to check it's info. Or when you don't want to manually delete the level. We have optional comments loading in the game's settings, optional song download by default -- optional level download feels kinda missing.

If I haven't convinced you, you can ignore this suggestion, or consider implementing something similar -- auto level remover, for example.

**Comment by NewAlexGamer** [comment]
i actually do it a lot and waiting for the level to load just to see 1 comment is so stupid, for reference it takes me 10 seconds to load any level that has 80k+ objects, imagine higher object heavy levels

**Comment by Brittank88** [comment]
What about a cancel button inside the level loading circle instead?

**Comment by Brittank88** [comment]
Or just rewriting the network systems used in GD to allow parallel network operations lol
